gpkit/interactive/plot_sweep1d.py

```diff
@@ -0,0 +1,48 @@
+"Implements plot_sweep1d function"
+import matplotlib.pyplot as plt
+import numpy as np
+from ..exceptions import InvalidGPConstraint
+from ..tools import autosweep_1d
+
```

Comment:
I can think of use cases that would involve sweeping a model, scripting something involving the result, and also plotting. For that reason I support separating the sweeping/solving from the plotting.